Readability.js

```diff
@@ -624,10 +623,10 @@ Readability.prototype = {
         // Move all of the page's children into topCandidate
         topCandidate = doc.createElement("DIV");
         neededToCreateTopCandidate = true;
-        var children = page.childNodes;
-        while (children.length) {
-          this.log("Moving child out:", children[0]);
-          topCandidate.appendChild(children[0]);
+        var kids = page.childNodes;
```

Comment:
Because all the text nodes get stuck in the topCandidate too, which is important if, for instance, we just have a <body> with a bunch of text in it.

I don't know how much we care about that case, though...